.github/workflows/benchmark.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
we no longer distribute `chia-blockchain` for python 3.7, so testing against it here pulls down an old version (1.8.2) which has a slightly different interface to `Program` (it doesn't have the `_run()` method).